tests/wicked/2nics/ref/t03_setup_second_card.pm

```diff
@@ -16,13 +16,18 @@ use base 'wickedbase';
 use strict;
 use warnings;
 use testapi;
+use lockapi;
+use utils 'systemctl';
 
 
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
     my $ip_address = $self->get_ip(type => 'second_card');
     assert_script_run("ip a a $ip_address dev " . $ctx->iface());
+    $self->get_from_data('wicked/dhcp/dhcpd_2nic.conf', '/etc/dhcpd.conf');
```

Comment:
so in case two nics scenario we downloading `wicked/dhcp/dhcpd.conf` which we actually don't need at all and starting DHCP with it and later we downloading the right config and restarting server . Maybe we would just download right config in before_test instead ? I think better to modify logic in before_test which download dhcp config and add special condition for 2nic scenario

Reply:
Do you mean checking `WICKED` for 2nic and  use the right dhcp.conf?

Reply:
exactly